### PR TITLE
Update config based on spec changes

### PR DESCRIPTION
**Pull request by crosbymichael**
For a full list of the changes see the commits and read the descriptions.

### CODE REVIEW

**Comment by mrunalp** [comment]
Looking...

File: README.md [hunk 1]
```diff
@@ -42,60 +42,28 @@ user named `daemon` defined within that file-system.
...
-            "cwd": ""
-        }
-    ],
+    "version": "0.1.1",
+    "platform": {
+        "os": "linux",
+        "arch": "amd64"
+    },
+    "process": {
+        "tty": true,
```
Comment (mrunalp) [inline README.md]: tty -> terminal

**Comment by mrunalp** [comment]
Needs rebase

**Comment by crosbymichael** [comment]
After rebase I removed the `all` from `make install`.  This is not how makefiles work. You don't run install and it rebuilds everything again and I cannot even use the Makefile anymore because you usually type sudo make install to get the binary in your system path and it cannot find go or godeps anymore because of this.

**Comment by mrunalp** [comment]
LGTM

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**